installing_on_kubernetes/_topics/prepare_kubernetes_namespace.md

```diff
@@ -7,6 +7,8 @@ The operator directory referenced below can be found [here]({{ site.data.product
    $ oc get crds | grep {{ site.data.product.operator_custom_resource_definition_name_plural }}.{{ site.data.product.operator_namespace }}
    $ oc create -f config/crd/bases/{{ site.data.product.operator_namespace }}_{{ site.data.product.operator_custom_resource_definition_name_plural }}.yaml
    ```
+   Prerequisite:
```

Comment:
I think this should be just below `## Preparing the Kubernetes namespace` since everything else in this doc is ours.  This is the only external dependency,

Reply:
thanks, moved it to the top